azure-mgmt-redis/src/main/java/com/microsoft/azure/management/redis/implementation/RedisAccessKeysInner.java

```diff
@@ -38,9 +38,9 @@ public String primaryKey() {
      * Set the primaryKey value.
      *
      * @param primaryKey the primaryKey value to set
-     * @return the RedisListKeysResultInner object itself.
+     * @return the RedisAccessKeysInner object itself.
      */
-    public RedisListKeysResultInner withPrimaryKey(String primaryKey) {
+    public RedisAccessKeysInner withPrimaryKey(String primaryKey) {
```

Comment:
Most probable we are not exposing this, if yes then we should remove it as you can send in keys most probably (but I could be wrong, usually services don't give capability to set the keys you can get or regenerate).
If this is only internal, ask log a bug with swagger that these properties are read only (I am assuming we are generating withPrimary.. withSecon.. as these properties are not read only)